core/embed/util/rsod/rsod.c

```diff
@@ -57,9 +58,16 @@ void rsod_terminal(const systask_postmortem_t* pminfo) {
       message = message_buf;
       break;
     case TASK_TERM_REASON_ERROR:
-      title = pminfo->error.title;
-      message = pminfo->error.message;
-      footer = pminfo->error.footer;
+      if (strnlen(pminfo->error.title, sizeof(pminfo->error.title)) != 0) {
```

Comment:
`pminfo->error->xxx` strings are always null-terminated, so we can simplify the condition to `if (pminfo->error.title[0] != '\0') {`.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/cdb0427a58d6413d64b94adb54bc2107f91cb249

Reply:
changed the other half here: https://github.com/trezor/trezor-firmware/commit/53d6f1f22bb9a8549c6f84ec6e6817054cf406b7